app/src/main/res/values/strings.xml

```diff
@@ -288,4 +288,7 @@
   <string name="topic_in_progress">Topic in Progress</string>
   <string name="stories_completed">Stories Completed</string>
   <string name="story_completed">Story Completed</string>
+  <!-- WalkthroughActivity -->
+  <string name="walkthrough_welcome_description">Learn new math skills with stories that show you how to use them in your daily life</string>
+  <string name="welcome">"Welcome "</string>
```

Comment:
Make this string similar to `topic_heading`, means use `%s` and update the kotlin code accordingly